docs/book/expressions.md

```diff
@@ -227,6 +227,60 @@ askName
     @name = $name
 ```
 
+## DEPLOY Variables
+
+DEPLOY variables are a type of memory storage variable that you can define as
+part of your Litexa configuration to exist at compilation time. They can be
+used like the other types of variables, but can also be used in [`when`
+statements](/reference/#when) and [file exclusion statements](/reference/#exclude-file).
+
+To define DEPLOY variables, add a `DEPLOY` object in your deployment targets
+like so:
+
+```javascript
+const deploymentConfiguration = {
+  name: 'cats-in-space',
+  deployments: {
+    development: {
+      ... // other configuration
+      DEPLOY: { // your DEPLOY variables go in this object
+        debug: true,
+        mode: 'practice'
```

Comment:
Would recommend casing these deploy properties consistently (lower case here, fully upper case in file exclusion example). Personally favor the full upper case since these should typically treated as (static) constants, but either way's fine.